packages/core/src/components/fields/ArrayField.tsx

```diff
@@ -478,6 +476,7 @@ class ArrayField<
           onBlur,
           onFocus,
           rawErrors,
+          totalItems: keyedFormData.length,
```

Comment:
is it possible for keyedFormData to be null or undefined?

Reply:
No, it is always at least an empty array